references/validationFunctions/exposure/checkAlcoholHistory.js

```diff
@@ -31,19 +31,40 @@ const validation = () =>
       
       // checks for a string just consisting of whitespace
       const checkforEmpty = (entry) => {return /^\s+$/g.test(decodeURI(entry).replace(/^"(.*)"$/, '$1'))};
-      alcoholHistoryCategories = ["daily drinker", "occasional drinker (< once a month)", "social drinker (> once a month, < once a week)", "weekly drinker (>=1x a week)"];
-      noOrUnknownAllowedCategories = ["none", "occasional drinker (< once a month)", "unknown"];
-       
-      if ($row.alcohol_consumption_category && $row.alcohol_consumption_category != null && !(checkforEmpty($row.alcohol_consumption_category))) {
-         alcoholConsumptionCategory = $row.alcohol_consumption_category.trim().toLowerCase();
-         if ((!$field || $field == null || checkforEmpty($field)) && alcoholHistoryCategories.includes(alcoholConsumptionCategory)) {
-            result = {valid:false, message: `If the donor is a '${alcoholConsumptionCategory}', then the 'alcohol_history' field must be submitted as well.`};
-         }
-         if ((!$field || $field === null || checkforEmpty($field) || $field.trim().toLowerCase() === 'no') && (!(noOrUnknownAllowedCategories.includes(alcoholConsumptionCategory)))) {
-            result = {valid:false, message: `If the donor is a '${alcoholConsumptionCategory}', then the 'alcohol_history' field must be submitted as 'Yes'.`};
-         }
+      const alcoholHistoryYes = ["daily drinker", "social drinker (> once a month, < once a week)", "weekly drinker (>=1x a week)"];
```

Comment:
wondering if the `occasional drinker` value should be here?

Reply:
No issue, don't need to add that value because it takes into account `alcohol_history` needs to be `Yes`